src/SFA.DAS.EAS.Web/Views/Shared/_Layout.cshtml

```diff
@@ -2,6 +2,7 @@
 @using SFA.DAS.Authorization
 @using SFA.DAS.Authorization.Mvc
 @using SFA.DAS.EAS.Web
+@using SFA.DAS.EAS.Web.Extensions

```

Comment:
Is this really part of this change?

Reply:
No, I made the change as part of a branch this is based on because it was causing thousands of prod errors and it annoyed me.